### PR TITLE
Single owner can manage delegates to mint

### PR DESCRIPTION
**Pull request by edgeoftheworld**
removed voting functions, single owner who manages minters, owner can change faucet contract on token

### CODE REVIEW

**Comment by edgeoftheworld** [comment]
here is the etherscan link to the deployed contract https://rinkeby.etherscan.io/address/0x29686429aad2b0674ed36a4d95b66c299dafc645

**Comment by fubuloubu** [comment]
Some update suggestions:
1. Make the delegate data structure into a map e.g. `mapping (address => bool) public canMint`
NOTE `public` creates a getter:
```
// Automatically generated by solidity:
function canMint(address _delegate) public view returns (bool) {
    return canMint[_delegate];
}
```
2. `owner` can control delegate's minting abilities (through `canMint`)
3. Add an event to track changes to access rights (for UI) e.g:
`event AccessChanged(address delegate, bool canMint)`
4. Set `canMint` of `owner` to true in the constructor

**Comment by edgeoftheworld** [comment]
updated contract on Rinkeby.  https://rinkeby.etherscan.io/address/0x5f1b069d88d239a52f51a6a7c5ecdfeec0afcbff

File: contracts/DappDevMint.sol [hunk 1]
```diff
@@ -88,94 +58,40 @@ contract DappDevMint
...
+
+  /**
+   * @dev Function add minters/delegates
+   * @param newDelegate The address that will be allowed to mint tokens.
+   */
+   function addDelegate(address newDelegate) public {
+      //only owner can add delegate
+      require(isOwner(msg.sender));
+      canMint[newDelegate] = true;
+      emit AccessControl(newDelegate, true);
```
Comment (fubuloubu) [inline contracts/DappDevMint.sol]: This is a new feature for v0.4.21, make sure the pragma reflects this

Comment (fubuloubu) [reply contracts/DappDevMint.sol]: “‘emit’ keyword in Solidity” @AniketEngg https://medium.com/@aniketengg/emit-keyword-in-solidity-242a679b0e1a

**Review by fubuloubu — CHANGES_REQUESTED** [review]

**Comment by fubuloubu** [comment]
Looking good! Will have to test next week!

**Comment by edgeoftheworld** [comment]
compiler version changed.
https://rinkeby.etherscan.io/address/0x616f47efc6dcd6ca09c08426c7111c51a5dd2d10

**Pull request merged**